src/v3/src/components/ReminderPrompt/ReminderPrompt.tsx

```diff
@@ -50,10 +51,8 @@ const ReminderPrompt: UISchemaElementComponent<{
     sessionStorage.setItem(RESEND_TIMESTAMP_SESSION_STORAGE_KEY, token);
   };
 
-  // eslint-disable-next-line arrow-body-style
-  const getResendTimestamp = (): string | null => {
-    return sessionStorage.getItem(RESEND_TIMESTAMP_SESSION_STORAGE_KEY);
-  };
+  const getResendTimestamp = (): string => sessionStorage
+    .getItem(RESEND_TIMESTAMP_SESSION_STORAGE_KEY) as string;
```

Comment:
This isn't specifically related to this change at all but I missed on the review for this PR that we cannot rely on `string` type after `getItem`. Later if we `parseInt` on the value here and it's `null` it will return `NaN`, which is probably not what we want to do math with. Let's set a sensible default in case it's not set and also put this return type back to `string | null`. @leemchale-okta